spec/swagger.yaml

```diff
@@ -185,6 +188,13 @@ parameters:
       - text/csv
     default: application/json
     description: The response media type
+  collectionEventTypes:
+    name: eventTypes
```

Comment:
Maybe it's late but what about `eventTypesFilter` instead ?
I remember we have something similar for webhooks and I think it make more sense when we send an empty `eventTypesFilter` by default, which mean we should not filter by any types.
Thoughts?

Reply:
We have `eventType` and `mediaType` in User API repo (those are strings and not arrays, but still do similar function), so calling actual variable `eventTypes` here is fine by me. @pmeth ?

Reply:
I understand the reasoning, but I think it's clear enough to see that if you omit it you get everything. I can't see why anyone would pass an empty array in any case. Also, since it's an array, the language of `eventTypesFilter` seems odd.